### PR TITLE
fixed email resend, profile, and signup.

### PR DESCRIPTION
**Pull request by jeromes123**
**OVERVIEW**
Fixed the bug where the user is unable to resend email verification. Fixed bug for signing up. Fixed bug for updating profile information.

### CODE REVIEW

File: src/app/Http/Controllers/SignupController.php [hunk 1]
```diff
@@ -86,11 +86,13 @@ public function verifyEmail($id, $hash): View
      */
     public function sendVerificationNotification(Request $request): view
```
Comment (mikco-hipe) [inline src/app/Http/Controllers/SignupController.php]: transfer this to service layer

File: src/resources/views/components/auth/verify-email.blade.php [hunk 2]
```diff
@@ -33,6 +33,9 @@
         <p>Resend Verification Email in <span id="countdown"></span> seconds</p>
         <form method="GET" action='/resend-email'>
             @csrf
+            <p>Your Email: </p><br>
+            <input type="email" value="{{ $useremail }}" disabled>
```
Comment (mikco-hipe) [inline src/resources/views/components/auth/verify-email.blade.php]: camel case for variable

**Review by mikco-hipe — APPROVED** [review]
LGTM 🙇

**Review by sunkenquest — APPROVED** [review]
LTFRB

**Pull request merged**